Lib/test/test_tools/test_pathfix.py

```diff
@@ -39,30 +39,62 @@ def test_pathfix(self):
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python',
-                ['-i', '/usr/bin/python3',]),
+                ['-i', '/usr/bin/python3']),
             '#! /usr/bin/python3',
         )
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python -R',
-                ['-i', '/usr/bin/python3', ]),
+                ['-i', '/usr/bin/python3']),
             '#! /usr/bin/python3',
         )
 
     def test_pathfix_keeping_flags(self):
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python -R',
-                ['-i', '/usr/bin/python3', '-k',]),
+                ['-i', '/usr/bin/python3', '-k']),
             '#! /usr/bin/python3 -R',
         )
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python',
-                ['-i', '/usr/bin/python3', '-k',]),
+                ['-i', '/usr/bin/python3', '-k']),
             '#! /usr/bin/python3',
         )
 
+    def test_pathfix_adding_flag(self):
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python',
+                ['-i', '/usr/bin/python3', '-a', 's']),
+            '#! /usr/bin/python3 -s',
+        )
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python -s',
+                ['-i', '/usr/bin/python3', '-a', 's']),
+            '#! /usr/bin/python3 -s',
```

Comment:
I would prefer to use different flags, like -s and -a S. Otherwise, it's unclear to see which one wins.